Lib/test/test_urllib.py

```diff
@@ -402,6 +402,8 @@ def test_read_bogus(self):
 Content-Type: text/html; charset=iso-8859-1
 ''')
         try:
+            # Silence destructor error
+            http.client.HTTPConnection.close = lambda self: None
```

Comment:
This will affect *all* instances of `HTTPConnection`.

Reply:
`fakehttp` replaces `http.client.HTTPConnection` with `FakeHTTPConnection` and `unfakehttp` restores the old value of `http.client.HTTPConnection` at the end of test. So I guess it's changing close on `FakeHTTPConnection` which is generated per test.

https://github.com/python/cpython/pull/13317/files#diff-618652ed25a24a1b635417b4264c9533L101

Reply:
I think @tirkarthi is right.

Reply:
Why not adding a close() method to FakeHTTPConnection?

Reply:
There are no test case failures on changing close in FakeHTTPConnection but I haven't had time to analyze it well to see if there are any other cases it affects. So I kept the change minimal to have reported tests silenced.

Reply:
After I wrote a similar change, I now understand this change and I agree that it's correct. But the change is surprising :-) I wrote PR #13996 which adds a "mock_close" attribute to fake_http(): if mock_close is true, a close() method which does nothing is defined. IMHO it's less surprising, since it's more explicitly that only the mock is affected.